chapter-06/start/Objects/Base/BaseGameObject.cs

```diff
@@ -11,11 +11,18 @@ public class BaseGameObject
 
         public int zIndex;
 
+        public int Width { get { return _texture.Width; } }
+        public int Height { get { return _texture.Height; } }
+        public Vector2 Position 
+        { 
+            get { return _position; } 
```

Comment:
Expression Body here is style++ :)

Reply:
eh, having the get and the set on the same line was just ugly